cli/main.go

```diff
@@ -13,5 +13,7 @@ func main() {
 	//	fmt.Println(err)
 	//}
 
+	slices.Register([]string { "1841289615922336", "76561197976367183", "76561198057633471", "76561198002272597", "76561197974723967", "76561197969022064"})
```

Comment:
I'd be nice if this was actually `map[string]string` I think. It is nice to know the human(slack) user name associated with these things
lets you tag people, etc

Reply:
Yeah, in the PR for the bot I added usernames as comments.
When the client fetches these, it returns usernames.
I didn't see a strong need to encode comments into the parameters we pass in.